pkg/runner/runner.go

```diff
@@ -192,35 +188,38 @@ func Run(cfg *Config) error {
 			return err
 		}
 
-		go func() {
-			log.Info("msg", fmt.Sprintf("Start listening for OTLP GRPC server on %s", cfg.OTLPGRPCListenAddr))
-			listener, err := net.Listen("tcp", cfg.OTLPGRPCListenAddr)
-			if err != nil {
-				log.Error("msg", "Listening for OTLP GRPC server failed", "err", err)
-				return
-			}
-
-			log.Info("msg", "Start OTLP GRPC server")
-			if err := grpcServer.Serve(listener); err != nil {
-				log.Error("msg", "Starting the OTLP GRPC server failed", "err", err)
-				return
-			}
-		}()
+		group.Add(
+			func() error {
+				listener, err := net.Listen("tcp", cfg.OTLPGRPCListenAddr)
+				if err != nil {
+					log.Error("msg", "Listening for OTLP GRPC server failed", "err", err)
+					return err
+				}
+				log.Info("msg", "Started OpenTelemetry OTLP GRPC server", "listening-port", cfg.OTLPGRPCListenAddr)
+				return grpcServer.Serve(listener)
+			}, func(err error) {},
+		)
 	}
 
 	mux := http.NewServeMux()
 	mux.Handle("/", router)
 
-	if cfg.TLSCertFile != "" {
-		err = http.ListenAndServeTLS(cfg.ListenAddr, cfg.TLSCertFile, cfg.TLSKeyFile, mux)
-	} else {
-		err = http.ListenAndServe(cfg.ListenAddr, mux)
-	}
+	group.Add(
+		func() error {
+			var err error
+			log.Info("msg", "Started Prometheus HTTP server", "listening-port", cfg.ListenAddr)
```

Comment:
Since we are not starting Prometheus HTT server consider changing text to: `Started Prometheus remote storage HTTP server`